query/iam/iam_no_inline_policy.sql

```diff
@@ -0,0 +1,40 @@
+select
```

Comment:
@Subhajit97 Can you please rename this query to `iam_group_no_inline_policies`?

Reply:
@cbruno10 This query checks inline policies for IAM users, groups and roles together.
Should we mention `iam_group` in query fileName, or we can add something that matches all three resource types?

Reply:
@Subhajit97 How about `iam_group_user_role_no_inline_policies.sql`?